aws_batch/indexing.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+import os
+import subprocess
+
+INPUT_FASTA_S3 = 's3://czbiohub-infectious-disease/references/mosquitos/mosquito_genomes2.fa'
+OUTPUT_PATH_S3 = 's3://czbiohub-infectious-disease/references/mosquitos'
+
+ROOT_DIR = '/mnt'
+DEST_DIR = ROOT_DIR + '/idseq/indexes' # generated indexes go here
```

Comment:
indices?

Reply:
In this context people seem to say 'indexes' more often (e.g. STAR manual)

Reply:
ok then!